# HG changeset patch
# User Rik <rik@octave.org>
# Date 1382889664 25200
#      Sun Oct 27 09:01:04 2013 -0700
# Node ID a230b996ae8a5c05ff2accb180f4ba1b873eda32
# Parent  7900595f1533adc9881a77ed1b565b7bb881e86d
Put dldfcn CPPFLAGS ahead of AM_CPPFLAGS (bug #36064)

* libinterp/dldfcn/config-module.awk: Put dldfcn CPPFLAGS ahead of AM_CPPFLAGS.

diff --git a/libinterp/dldfcn/config-module.awk b/libinterp/dldfcn/config-module.awk
--- a/libinterp/dldfcn/config-module.awk
+++ b/libinterp/dldfcn/config-module.awk
@@ -61,17 +61,17 @@ BEGIN {
     sub (/\.cc$/, "", basename);
     print "";
     printf ("dldfcn_%s_la_SOURCES = dldfcn/%s\n",
 	    basename, files[i]);
     if (cppflags[i])
       {
         printf ("dldfcn/%s.df: CPPFLAGS += %s\n",
                 basename, cppflags[i]);
-        printf ("dldfcn_%s_la_CPPFLAGS = $(AM_CPPFLAGS) %s\n",
+        printf ("dldfcn_%s_la_CPPFLAGS = %s $(AM_CPPFLAGS)\n",
                 basename, cppflags[i]);
       }
     printf ("dldfcn_%s_la_LDFLAGS = -avoid-version -module $(NO_UNDEFINED_LDFLAG) %s $(OCT_LINK_OPTS)\n",
             basename, ldflags[i]);
     printf ("dldfcn_%s_la_LIBADD = $(DLD_LIBOCTINTERP_LIBADD) ../liboctave/liboctave.la %s $(OCT_LINK_DEPS)\n",
             basename, libraries[i]);
   }
 }
